### ISSUE TITLE
Installation failed  ./scripts/install.sh: 20: ./scripts/install.sh: [[: not found

### ISSUE THREAD

**Issue by naseemkullah** [hayorov/helm-gcs#20]
Hello,

First off, congrats on taking over the repo!

So my pipeline to deploy helm charts is breaking since helm-gcs 0.2.1 with the following error:

```
Step #3: Installing helm-gcs 0.2.1 ...
Step #3: ./scripts/install.sh: 20: ./scripts/install.sh: [[: not found
Step #3: 
Step #3: gzip: stdin: not in gzip format
Step #3: tar: Child returned status 1
Step #3: tar: Error is not recoverable: exiting now
Step #3: helm-gcs 0.2.1 is correctly installed.
Step #3: 
Step #3: Init a new repository:
Step #3:   helm gcs init gs://bucket/path
Step #3: 
Step #3: Add your repository to Helm:
Step #3:   helm repo add repo-name gs://bucket/path
Step #3: 
Step #3: Push a chart to your repository:
Step #3:   helm gcs push chart.tar.gz repo-name
Step #3: 
Step #3: Update Helm cache:
Step #3:   helm repo update
Step #3: 
Step #3: Get your chart:
Step #3:   helm fetch repo-name/chart
Step #3: 
Step #3: Installed plugin: gcs
Step #3: /builder/home/.helm/plugins/helm-gcs/scripts/pull.sh: 3: /builder/home/.helm/plugins/helm-gcs/scripts/pull.sh: /builder/home/.helm/plugins/helm-gcs/bin/helm-gcs: not found
Step #3: Error: Looks like "gs://my-charts-bucket" is not a valid chart repository or cannot be reached: plugin "scripts/pull.sh" exited with error
Step #3: Adding repo my-repo gs://my-charts-bucket
Step #3: in helmfile.yaml: helm exited with status 1:
Step #3:   /builder/home/.helm/plugins/helm-gcs/scripts/pull.sh: 3: /builder/home/.helm/plugins/helm-gcs/scripts/pull.sh: /builder/home/.helm/plugins/helm-gcs/bin/helm-gcs: not found
Step #3:   Error: Looks like "gs://my-charts-bucket" is not a valid chart repository or cannot be reached: plugin "scripts/pull.sh" exited with error
Finished Step #3
```

Any ideas?

**Comment by helmecke** [comment]
Release 0.2.1 is missing it's downloads.

**Comment by hayorov** [comment]
Hello @naseemkullah  thank you for the quick response about v0.2.1.

I did not expect such a quick response, the version is still in pre.

I'll fix issue and setup install test.

Next time it will be in PR :)

@helmecke correct - it's `in pre`. I want to set up automatic releases with popular dists.

**Comment by merqurio** [comment]
We experienced a similar issue in our CI system too

```sh
$ helm plugin install https://github.com/nouney/helm-gcs
Installing helm-gcs 0.2.1 ...
tar: invalid magic
tar: short read
helm-gcs 0.2.1 is correctly installed.

Init a new repository:
  helm gcs init gs://bucket/path

Add your repository to Helm:
  helm repo add repo-name gs://bucket/path

Push a chart to your repository:
  helm gcs push chart.tar.gz repo-name

Update Helm cache:
  helm repo update

Get your chart:
  helm fetch repo-name/chart

Installed plugin: gcs
$ helm repo add iomed gs://helm-iomed-repository
/iomed/datascience/plugins/helm-gcs/scripts/pull.sh: line 3: /iomed/datascience/plugins/helm-gcs/bin/helm-gcs: not found
Error: Looks like "gs://helm-iomed-repository" is not a valid chart repository or cannot be reached: plugin "scripts/pull.sh" exited with error
```

**Comment by hayorov** [comment]
@merqurio and all folks I'm really sorry for this breaking change.
I reverted default version to 0.2.0 till 0.2.1 will be ready.

```
allexx@allexxmbpn:~/alexk/github.com/hayorov/helm-gcs|master⚡
⇒  helm plugin install https://github.com/nouney/helm-gcs
Installing helm-gcs 0.2.0 ...
x LICENSE
x README.md
x helm-gcs
helm-gcs 0.2.0 is correctly installed.

Init a new repository:
  helm gcs init gs://bucket/path

Add your repository to Helm:
  helm repo add repo-name gs://bucket/path

Push a chart to your repository:
  helm gcs push chart.tar.gz repo-name

Update Helm cache:
  helm repo update

Get your chart:
  helm fetch repo-name/chart

Installed plugin: gcs
```

**Comment by hayorov** [comment]
Fixed in #21
